dashboard/app/views/plc/course_units/launch.html.haml

```diff
@@ -0,0 +1,21 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/plc', media: 'all'
+
+%h1 Launch PLC Courses
+
+= form_tag(action: 'launch_plc_course') do
+  %div#notice{style: '-webkit-user-select: initial'}
+    = params[:notice].try(:html_safe)
+
+  .field
+    = label_tag 'Pick a Course to Launch'
+    = select_tag :plc_course_unit_id, options_for_select(options_for_plc_course_launch)
```

Comment:
One other question for my understanding...this is some Rails magic, do you happen to know how this view knows about this method definition??

Reply:
Just did a little experiment with a random view outside of PLC...apparently you can call this method in any view! and it will work:
https://www.rubyguides.com/2020/01/rails-helpers/

Reply:
this is the magic and challenge of rails... everything is available everywhere, which is convenient, but makes it very hard to encapsulate anything in order to control how it is accessed.

Reply:
Thank you both for explaining and digging in. I sometimes just use the try and see method and don't always know the why.  @bencodeorg I really have appreciated your thoughtful reviews this week